code/datums/components/tackle.dm

```diff
@@ -299,6 +301,8 @@
 		attack_mod -= 2
 	if(HAS_TRAIT(sacker, TRAIT_DWARF))
 		attack_mod -= 2
+	if(HAS_TRAIT(target, TRAIT_TINY))
```

Comment:
```suggestion
	if(HAS_TRAIT(sacker, TRAIT_TINY))
```

Reply:
also, why is the tackling attack penalty for being tiny less than the tackling defence penalty for being tiny?

Reply:
Mistake on my part. changed so they're both 4